icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2437,15 +2437,31 @@ class U_I18N_API FormattedNumber : public UMemory, public FormattedValue {
      */
     FormattedNumber& operator=(FormattedNumber&& src) U_NOEXCEPT;
 
-    /** @copydoc FormattedValue::toString() */
+    // Copybrief: this method is older than the parent method
```

Comment:
are the // comments necessary?

Reply:
These four cases of `/copybrief` will be the exception; the standard rule will be to use `/copydocs` instead.  I put the comments to explain why there is an exception.